label_studio/ml_models/models.py

```diff
@@ -142,3 +142,10 @@ def input_file_name(self):
     @property
     def output_file_name(self):
         return f'{self.project.id}_{self.model_version.pk}_{self.pk}/output_tasks.csv'
+
+    @property
+    def error_file_name(self):
+        return f'{self.project.id}_{self.model_version.pk}_{self.pk}/output_errors.csv'
+
+    # TODO should this be a CharField in case Adala changes its job ID format?
```

Comment:
the main thing it gives is validation. eg if you pass it in as a string with or without dashes, with or without the header, it will resolve to the same UUID. Seems worthwhile if we will stick with that format.

Reply:
I added this in 939 PR as well, and made it a charfield 
I figured we will have more job IDs than just Adala's in the future and it should be flexible

Reply:
changed to charfield